Makefile

```diff
@@ -56,6 +56,7 @@ lint:
 	  --global module \
 	  --global require \
 	  --global self \
+	  --rule 'indent: [off]' \
```

Comment:
`key-spacing` doesn't often cause problems, so disabling that rule selectively seems appropriate.